v2/kafka-common/src/main/java/com/google/cloud/teleport/v2/kafka/options/KafkaReadOptions.java

```diff
@@ -49,23 +49,49 @@ public interface KafkaReadOptions extends PipelineOptions {
 
   void setKafkaReadTopics(String inputTopics);
 
-  @TemplateParameter.Enum(
+  @TemplateParameter.Boolean(
       order = 3,
       groupName = "Source",
+      name = "enableCommitOffSets",
+      optional = true,
+      description = "Enable committing offsets to Kafka",
+      helpText =
+          "Enable KafkaIO to commit offsets after consuming the record. Enabling this requires you to provide"
+              + "a Consumer group ID or the template launcher will fail.")
+  @Default.Boolean(false)
+  Boolean getEnableCommitOffSets();
+
+  void setEnableCommitOffSets(Boolean value);
+
+  @TemplateParameter.Text(
+      order = 4,
+      groupName = "Source",
+      optional = true,
+      description = "Consumer group id to commit offsets",
```

Comment:
"Consumer group id to commit offsets" -> "Consumer Group ID" (keep it short and capitalized).

Reply:
Done